### DISCUSSION TITLE
followed tutorial; .next or .out not created

### DISCUSSION THREAD

**Discussion by karadahl** [flowershow/flowershow#460] [General and Help] [answered]
Hello, I've run npx flowershow@latest export and I see that a .contentlayer folder is created inside .flowershow, but not a .next or .out folder.  I've also tried flowershow@latest build. Did I miss a step somewhere?  The preview doesn't work either.

**Comment by olayway** [comment]
Hello, @karadahl! Thanks for reporting that! Can you please share the errors you're getting?

**Reply by karadahl** [reply]
If the markdown files are empty, everything works fine.  If there is text, even just the word "test" in index.md (or any other .md) then the export (or preview) stops right after contentlayer build, with no error whatsoever.

**Comment by karadahl** [comment]
Hi, thanks for responding so quickly.  There are no errors that I can tell.  The preview (in browser) gives "connection refused" which I expect since the preview returned me to the command line.

![Screenshot 2023-03-24 at 8 08 06 AM](https://user-images.githubusercontent.com/32492414/227529483-714f80e5-d6dc-4919-b16a-791887d6fe60.png)

<img width="771" alt="Screenshot 2023-03-24 at 8 07 45 AM" src="https://user-images.githubusercontent.com/32492414/227529419-0a179b1a-094b-41b4-8d25-dffd94aa43cd.png">


karadahl@multipass .flowershow % ls -la
total 776
drwxrwxr-x   26 karadahl  staff     832 Mar 24 07:55 .
drwxr-xr-x    7 karadahl  staff     224 Mar 24 07:54 ..
drwxr-xr-x    3 karadahl  staff      96 Mar 24 07:55 .contentlayer
-rw-r--r--    1 karadahl  staff     274 Mar 24 07:07 CHANGELOG.md
-rw-r--r--    1 karadahl  staff     213 Mar 24 07:07 README.md
drwxrwxr-x    3 karadahl  staff      96 Mar 24 07:54 components
drwxrwxr-x    3 karadahl  staff      96 Mar 24 07:54 config
lrwxr-xr-x    1 karadahl  staff      23 Mar 24 07:54 content -> ../digital-garden-files
-rw-r--r--    1 karadahl  staff    7357 Mar 24 07:07 contentlayer.config.ts
-rw-r--r--    1 karadahl  staff     126 Mar 24 07:07 giscus.json
drwxrwxr-x    3 karadahl  staff      96 Mar 24 07:54 layouts
drwxrwxr-x    4 karadahl  staff     128 Mar 24 07:54 lib
-rw-r--r--    1 karadahl  staff     201 Mar 24 07:07 next-env.d.ts
-rw-r--r--    1 karadahl  staff     255 Mar 24 07:07 next.config.mjs
drwxr-xr-x  545 karadahl  staff   17440 Mar 24 07:55 node_modules
-rw-r--r--    1 karadahl  staff  341735 Mar 24 07:55 package-lock.json
-rw-r--r--    1 karadahl  staff    1873 Mar 24 07:07 package.json
drwxrwxr-x    5 karadahl  staff     160 Mar 24 07:54 pages
-rw-r--r--    1 karadahl  staff     204 Mar 24 07:07 postcss.config.js
drwxrwxr-x    4 karadahl  staff     128 Mar 24 07:54 public
drwxrwxr-x    5 karadahl  staff     160 Mar 24 07:54 scripts
drwxrwxr-x    5 karadahl  staff     160 Mar 24 07:54 styles
-rw-r--r--    1 karadahl  staff    1314 Mar 24 07:07 tailwind.config.cjs
-rw-r--r--    1 karadahl  staff     984 Mar 24 07:07 tsconfig.json
-rw-r--r--    1 karadahl  staff      82 Mar 24 07:07 tsconfig.mdx.json
-rw-r--r--    1 karadahl  staff      47 Mar 24 07:07 types.d.ts
karadahl@multipass .flowershow %

**Reply by rufuspollock** [reply]
@karadahl we are looking at this and trying to figure out the issue. Do keep us posted if you make any progress.

**Comment by olayway** [comment, marked as answer]
@karadahl can you please try re-installing Flowershow template with `npx flowershow@latest install` and run preview and/or export commands again?

**Reply by karadahl** [reply]
that worked.  thank you!

**Comment by ruairimcn** [comment]
Hey, I'm having this issue also. Neither 'preview' or 'export' works, and there are no error or success messages given in Terminal. Using latest versions of Node, Mac OS X, Obsidian, and Flowershow. Have tried reinstalling but that didn't work for me.
![Screenshot 2023-06-04 at 6 14 09 p m](https://github.com/datopian/flowershow/assets/103217833/019bc558-5f6b-4b18-91cd-b01ef6003099)

**Comment by olayway** [comment]
Closing as this was related to the old, self-hosted Flowershow version
